lizadaly/manicule
# structure

- Move old stuff out of the way
- Add things
- Handle metadata
- Test this
- Delete tests for now
- Use cache
- Switch away from import
- Don't use a public dir
- Update image
- Add cacheable image
- Add basic page engine
- Use navstrip to switch between pages
- Add web workers for caching
- Add next/previous button nav
- Add fonts
- Add better CSS
- Different button style
- Clean up style for opener button
- Improve toggle
- Add primitive image zooming/panning
- Add reset handler
- Add second model
- Use natural sizes for faster loading
- Add documentation to the home page
- Add loading spinner to leaves
- Don't collect images we can't yet see
- Update docs
- Add viscoll docs
- Docs
- Structure view
- Add structure lines
